client/app/components/dialog-editor-boxes/dialog-editor-boxes.html

```diff
@@ -0,0 +1,57 @@
+<div class='dialog'>
+  <div style="background-color: #f1f1f1; padding: 20px;"
+       ng-model='dialogEditorBoxes.dialogTabs[dialogEditorBoxes.service.activeTab].dialog_groups'
+       ui-sortable='dialogEditorBoxes.sortableOptionsBox'
+       ng-repeat='tab in dialogEditorBoxes.dialogTabs'
+       ng-show='tab.position == dialogEditorBoxes.service.activeTab'>
+    <!-- tab content -->
+    <div ng-repeat='box in tab.dialog_groups track by $index'>
+      <div class="panel panel-default"
+           data-drop='true'
+           jqyoui-droppable='{multiple: true, onDrop: "dialogEditorBoxes.droppableOptions"}'
+           ng-model='box.dialog_fields'>
+        <div class="panel-heading">
+          <i class="pficon-edit"
+             ng-click='dialogEditorBoxes.editDialogModal(
+               dialogEditorBoxes.service.activeTab,
+               box.position
+             )'></i>
+          <strong>{{ box.label }}</strong>
+          <button type="button" class="close"
+                  ng-click="dialogEditorBoxes.removeBox(box.position)">
+            <span aria-hidden="true">
+              <i class="fa fa-times"></i>
+            </span>
+          </button>
+        </div>
+        <div class="panel-body">
+          <div ui-sortable='dialogEditorBoxes.sortableOptionsFields'
+               ng-model='box.dialog_fields'>
+            <div class="well well-lg nosort"
```

Comment:
the icon will throw off the translation here, best enclose text as {{ 'Drag your components here` | translate }}